index.html

```diff
@@ -0,0 +1,63 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>?Tamoor's Portfolio</title>
+  <link rel="stylesheet" href="style.css"/>
+</head>
+<body class="body">
+  <header>
+    <nav class="head">
+      <a href="#" class="logo">My logo</a>
+  <img src="images/Union.png" class="nav-bar">
+      </nav>
+      </header>
+     <main class="container">
+        <div class="background">
+    <section class="home-section">
+      <h1 class="header">
+        I'm Tamoor<br>
+        Glad to see <br>
+        you!
+    </h1>
+      <p class="text">
+        I’m a software developer! I can help you build a product ,feature or website Look through some of my work and experience! If you like what you see and have a project you need coded ,
+        don’t hestiate to contact me.
+    </p>
+<a href="#" class="link">LET'S CONNECT</a>
```

Comment:
- [OPTIONAL] I highly recommend you use the heading `hx` tag on this `LET'S CONNECT` subtitle in order to follow the HML best practices because this is is not a link.

Reply:
@levy002 Border radius has been updated and added to the background!